Dockerfile

```diff
@@ -1,6 +1,13 @@
-FROM node:lts
-RUN apt-get update
-RUN apt-get install dos2unix
-RUN yarn global add azurite
-RUN dos2unix /usr/local/share/.config/yarn/global/node_modules/azurite/bin/azurite
-ENTRYPOINT azurite
\ No newline at end of file
+FROM mcr.microsoft.com/azure-functions/node:2.0
+
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
+ENV PATH=$PATH:/home/node/.npm-global/bin
+WORKDIR /home/node
+ENV AzureWebJobsScriptRoot=/home/node
+COPY package.json /home/node
+# RUN npm i -g copyfiles onchange rimraf typescript ttypescript
```

Comment:
Is this here for future reference, or a leftover from local development?

Reply:
Oh yes! let's update the readme. Now we only need two steps, right?

1. Download and install docker
2. docker-compose up

Well... that's what I'm thinking on running when testing this PR later today. @davisokoth let me know if there's anything else to do

Reply:
@Crysfel I should probably add a `docker-compose.yml` file so that its that simple. Will commit that tomorrow

Reply:
Will update `README.md` as well and fix the conflicts on `settings.json`